src/components/Checkout/LoanPrice.vue

```diff
@@ -81,24 +85,28 @@ export default {
 		prices() {
 			if (this.type === 'loan') {
 				// determine how many (if any) overall additional shares are remaining
-				let remainingShares = parseFloat(this.loanAmount) - parseFloat(this.fundedAmount);
+				let remainingAmount = parseFloat(this.loanAmount) - parseFloat(this.fundedAmount);
 
 				// subtract reservedAmount shares (minus our own reserved shares)
 				// - only do this for loans that are not ending soon
 				// - for loans ending soon we just show remaining shares which are all un-reserved
 				if (!this.isExpiringSoon) {
-					remainingShares -= (parseFloat(this.reservedAmount) - parseInt(this.price, 10));
+					remainingAmount -= (parseFloat(this.reservedAmount) - parseFloat(this.price, 10));
 				}
 
-				// if we've met reserve ensure atleast this loan share is set
-				if (remainingShares < parseInt(this.price, 10)) remainingShares = parseInt(this.price, 10);
+				// if we've met reserve ensure at least this loan share is set
+				remainingAmount = Math.max(remainingAmount, parseFloat(this.price, 10));
 
-				// get count of shares based on available remaining shares
-				const sharesBelowReserve = parseInt(remainingShares, 10) / 25;
 				// convert this to formatted array for our select element
-				const reservePriceArray = this.buildShareArray(sharesBelowReserve);
-
-				return reservePriceArray;
+				const minAmount = parseFloat(this.minAmount || 25, 10); // 25_hard_coded
+				const numShares = remainingAmount / minAmount;
+				const priceArray = [];	// ex. priceArray = ['25', '50', '75']
+				// loop and build formatted array
+				// eslint-disable-next-line no-plusplus
+				for (let i = 1; i <= numShares; i++) {
```

Comment:
We can change this line to: 
`for (let i = 1; i <= numShares; i+=1)`
And remove the no-plusplus exception
Reference: https://eslint.org/docs/rules/no-plusplus

Reply:
Switched to `Array(N).map((_, i) => ...)`